src/test/log_basic/log_basic.c

```diff
@@ -171,9 +171,11 @@ printit(const void *buf, size_t len, void *arg)
 {
 	char *str = alloca(len + 1);
 
-	strncpy(str, buf, len);
-	str[len] = '\0';
-	UT_OUT("%s", str);
+	if (str) {
```

Comment:
alloca can't fail. If it overflows its undefined behavior.
Instead, I'd recommend changing this to a malloc.